site/data/charts/downloads.json

```diff
@@ -0,0 +1,71 @@
+{
+  "title": "Direct Downloads",
+  "description": "Direct downloads since v2.4.3. Its worth noting that downloads have reduced since the Docker images have become more popular.",
```

Comment:
```suggestion
  "description": "Direct downloads since v2.4.3. It's worth noting that downloads have reduced since the Docker images have become more popular.",
```

Reply:
Just a thought - I put the descriptions in the data files to keep everything together, but maybe they are more 'content' and so should be passed in from the md file as a parameter to the snippet - thoughts?

Reply:
I’m good with it either way. As long as it is consistent.

Reply:
OK, lets leave as is for now, in which case its ready for review again :)